src/main/java/core/basesyntax/BooleanExpression.java

```diff
@@ -14,6 +14,11 @@
      * без использования конструкций if-else</p>
      */
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+        return (a == true && b == true && c == false && d == false)
+                || (a == true && b == false && c == true && d == false)
+                || (a == true && b == false && c == false && d == true)
+                || (a == false && b == true && c == false && d == true)
+                || (a == false && b == true && c == true && d == false)
+                || (a == false && b == false && c == true && d == true) ? true : false;
```

Comment:
(a && b) is the same as (a == true && b == true) but code would be much shorter
and in general this solution might be simplified. 
try to use ^ (XOR) here